packages/crypto/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './jose.js';
 export * from './local-kms-crypto.js';
-export * as utils from './utils.js';
+import * as utils from './utils.js';
+export { utils };
```

Comment:
can you switch this back?  there's nothing wrong with this syntax

Reply:
I can't, unfortunately it's a constraint with api-extractor...

Reply:
https://github.com/microsoft/rushstack/issues/2780